Task 2/Task2.3/Trainings/Training.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace Task2._3.Trainings
+{
+    public class Training : TrainingEntity
+    {
+        private int _trainingsCount;
```

Comment:
_trainings not perfect name, better _lessons or _trainingEntitys

Reply:
Fixed